### PR TITLE
Simplify English used in guide

### PR DESCRIPTION
**Pull request by KGrewal1**
First draft of trying to simplify the English used in the guide section of the book, as well as normalising between 'you' for the reader and 'we' for the authors

### CODE REVIEW

**Comment by KGrewal1** [comment]
[See Zulip discussion](rust-lang.zulipchat.com/#narrow/channel/246057-t-cargo/topic/Difficult.20ESL.20words.20Cargo.20Book)

File: src/doc/src/guide/cargo-toml-vs-cargo-lock.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ We recommend pairing this with
 
 Let’s dig in a little bit more.
 
-`Cargo.toml` is a [**manifest**][def-manifest] file in which we can specify a
-bunch of different metadata about our package. For example, we can say that we
+`Cargo.toml` is a [**manifest**][def-manifest] file in which you can specify a
+bunch of different metadata about our package. For example, you can say that you
```
Comment (weihanglo) [inline src/doc/src/guide/cargo-toml-vs-cargo-lock.md:20]: Should we split pronoun switch into a separate PR? The purpose is quite different from using simple English and people may have different preference on it?

Comment (KGrewal1) [reply src/doc/src/guide/cargo-toml-vs-cargo-lock.md:20]: Sure - separated the commits so will be easy for me to split

Comment (KGrewal1) [reply src/doc/src/guide/cargo-toml-vs-cargo-lock.md]: Reverted the pronoun component

**Review by weihanglo — APPROVED** [review]

File: src/doc/src/guide/cargo-toml-vs-cargo-lock.md (hunk 1 above)
Comment (weihanglo) [reply src/doc/src/guide/cargo-toml-vs-cargo-lock.md]: Thanks!
